crates/mun_syntax/src/tests/parser.rs

```diff
@@ -186,3 +186,16 @@ fn loop_expr() {
     }"#,
     )
 }
+
+#[test]
+fn break_expr() {
+    ok_snapshot_test(
+        r#"
+    fn foo() {
+        break;
+        if break { 3; }
```

Comment:
Is this supposed to work? What is the meaning?

Reply:
Yeah, it works! Its a bit strange but also works with Rust.